installing/installing_platform_agnostic/installing-platform-agnostic.adoc

```diff
@@ -63,7 +63,7 @@ include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[lev
 
 To install {product-title} on infrastructure that you provision, you must install {op-system-first} on the machines. When you install {op-system}, you must provide the Ignition config file that was generated by the {product-title} installation program for the type of machine you are installing. If you have configured suitable networking, DNS, and load balancing infrastructure, the {product-title} bootstrap process begins automatically after the {op-system} machines have rebooted.
 
-If your infrastructure supports it, install {op-system} on the machines by following either the steps to use an ISO image or network PXE booting. In some cases, you might be able to upload an appropriate {op-system} image from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror] to your cloud provider and use that image to create the machines.
+If your infrastructure supports it, install {op-system} on the machines by following either the steps to use an ISO image or network PXE booting. In some cases, you might be able to upload an appropriate {op-system} image from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/[{op-system} image mirror] to your cloud provider and use that image to create the machines.
```

Comment:
Probably out of scope for this PR with it being so close to release but I'd think we could use the parent-level link to avoid the need to change this with each minor release: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/

Reply:
I've seen from 4.6 to 4.8 that the number of these links was reduced. I'll investigate this improvement post GA.